frontend/src/views/Reporting/ReportTemplates/FieldDetailSection.tsx

```diff
@@ -230,15 +217,17 @@ export default function CompleteReportTemplate({
         disableColumnMenu
         hideFooterPagination
         hideFooter
-        rowHeight={ROW_HEIGHT}
-        autoHeight
+        getRowHeight={() => 'auto'}
       />
 
       {booleanChecker(SoilTest) ? (
         <>
           <SectionTitle>Soil Test Information</SectionTitle>
           <FieldInfoItem style={{ marginBottom: '12px' }}>
-            <strong>Soil Test Results:</strong> {SoilTest?.sampleDate?.toString()}
+            <strong>Soil Test Results:</strong>{' '}
+            {SoilTest?.sampleDate
+              ? new Date(SoilTest.sampleDate).toLocaleDateString('sv-SE', { dateStyle: 'short' })
```

Comment:
What format is this?

Reply:
Date was being saved as ISOstring in the NMP file. This converts it to yyyy--mm-dd (2025-07-01)